packages/transport-ble/src/transports/BleOutboundTransport.ts

```diff
@@ -0,0 +1,42 @@
+import type { Peripheral } from '@animo-id/react-native-ble-didcomm'
+import type { Agent, Logger, OutboundPackage, OutboundTransport } from '@aries-framework/core'
+
+import { AriesFrameworkError } from '@aries-framework/core'
+
+export class BleOutboundTransport implements OutboundTransport {
+  public supportedSchemes: string[] = ['ble']
+  private peripheral: Peripheral
+  private logger!: Logger
+
+  public constructor(peripheral: Peripheral) {
+    this.peripheral = peripheral
+  }
+
+  public async start(agent: Agent): Promise<void> {
+    this.logger = agent.config.logger
+
+    agent.config.logger.debug('Starting BLE outbound transport')
+  }
+
+  public async sendMessage(outboundPackage: OutboundPackage): Promise<void> {
+    const { payload, endpoint } = outboundPackage
+
```

Comment:
same here (won't comment again), but adding new lines between each call just adds noice, instead of making the code cleaner to read. Group related calls (or that belong to the same category, so logs can be in a 'group') and add newlines between unrelated / new things